src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,144 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
```

Comment:
Ordering of the import statements should be consistent. 
It should be arranging all the java.io. first, then followed by java.util.